docs/server-side-configuration.md

```diff
@@ -0,0 +1,37 @@
+
+---
+
+# Cozy MyAccounts: Server side configuration
+
+## Authorized categories list
+
+The MyAccounts categories names list won't be automatically computed from available connectors in order to keep it clear and relevant for all Cozy users with a growing number of connectors.
+
+Instead, they need to be configured on the served side, in a simple way, from a config file. This is a list of categories authorized to be used:
+
+```javascript
+{
+    authorizedCategories:
+        {
+            'insurance': 1,
```

Comment:
@m4dz I just thought here about a list of authorized categories as an object with slug names as properties. This list is just used to lookup in, if a category is 'authorized' to be used in the client application or not. The `1` values are just to give a truthy value.

__Why an object?__ Actually for two main reasons, all related to the fact that this list will be used only to lookup in:

* Lookup in a list is more readable using an object than an array:

    ```coffee
    if not authorizedCategories[category]
        # do something

    # VS

    if authorisedCategories.indexOf(category) isnt -1
        # do something
    ```

    Even if this part will be solved with the ES7 `includes` function, there is another point.

* For performance reason. Use an object to lookup in thanks to property access is so much faster than lookup into an array using `indexOf`. You can test this performance with this [jsperf](https://jsperf.com/array-indexof-vs-object-key-lookup2). And the ES7 `includes` is not much faster for now ([jsperf](https://jsperf.com/lodash-includes-vs-indexof/2)).

Of course, it's my opinion and all of that is open to discussions.